tutorials/introductory/pyplot.py

```diff
@@ -259,7 +259,7 @@ def f(t):
 
 ###############################################################################
 # The `~.pyplot.figure` call here is optional because
-# ``figure(1)`` will be created by default, just as a ``subplot(111)``
+# ``figure(1)`` will be created by default, just as a ``subplot()``
```

Comment:
Does this need the figure(1)

Reply:
No. I've rewritten the sentence to make more sense. But don't get me started, otherwise I'll rewrite the whole tutorial 😝.